### PR TITLE
Tasty ham uat enforce seq on act

### PR DESCRIPTION
**Pull request by cp-ps**

### CODE REVIEW

File: misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx [hunk 1]
```diff
@@ -26,12 +26,35 @@ const RawMaterialIssueActivity = (props) => {
...
+      return false;
+    }
+    if (!isUserEditable) {
+      return true;
+    }
+    if (currentLine.completeStatus === CompleteStatus.COMPLETED) {
+      return false;
+    }
+    return previousLine && previousLine.completeStatus !== CompleteStatus.COMPLETED;
+  }, []);
```
Comment (teosarca) [inline misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx:40]: pls include isAlwaysAvailableToUser, isUserEditable to dependencies array.... or don't use useCallback

Comment (teosarca) [reply misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx:40]: ahh, i've read again the code... basically there is no point to for useCallback here.

**Review by teosarca — CHANGES_REQUESTED** [review]
I've reviewed the last 6 commits. pls check my comments.

File: misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx (hunk 1 above)
Comment (cp-ps) [reply misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/RawMaterialIssueActivity.jsx:40]: yes, you are right, it was a left over from a previous commit, thx for pointing it out

**Review by teosarca — APPROVED** [review]

**Pull request merged**